src/example_scripts/daemonize_workflow.ml

```diff
@@ -0,0 +1,52 @@
+#use "topfind"
+#thread
+#require "ketrew"
```

Comment:
I used to avoid that because I don't like having code in the repo that doesn't get compiled regularly.

Since they require the ocamlfind package `ketrew` *globally*, we should run those 2 scripts in the travis build.